joomla/joomla-cms
# [4.0] Align sidebar icon

Pull Request for Issue #27135 .

### Summary of Changes
Moves sidebar icons to using flex. Fixes alignment issue. 


### Testing Instructions
Apply this patch and run node build.js --compile-css for updating the changed SCSS.

Check sidebar icons are aligned.

### Before PR
![image](https://user-images.githubusercontent.com/2803503/69618845-7a086080-1032-11ea-9220-a73470d383f8.png)

### With PR
![image](https://user-images.githubusercontent.com/2803503/69618836-7674d980-1032-11ea-8705-f0e6925eea5b.png)

### Documentation Changes Required
